lib/packages/docDiffLogger.js

```diff
@@ -0,0 +1,60 @@
+var _ = require('lodash');
+var Package = require('../Package');
+var util = require('util');
+var diff = require('objectdiff').diff;
+var firstDocs, startDocs, lastDocs;
+var firstProcessor, startProcessor, lastProcessor;
```

Comment:
irstProcessor, startProcessor, lastProcessor : What are these three variables? They are assigned but not used, I do not see their utilities